libs/sdk-ui-tests-e2e/cypress/tools/dashboardHeader.ts

```diff
@@ -32,8 +49,16 @@ export class DashboardHeader {
         return this;
     }
 
+    getDashboardTitleElement() {
+        return cy.get(".s-gd-dashboard-title");
```

Comment:
I think we should define css as a const, make it easy to read and reuse.
ex: getDashboardTitleElement() and dashboardTitleHasValue() use the same cy.get(".s-gd-dashboard-title")